docs/schema/listtemplate-element-list-template.md

```diff
@@ -69,229 +70,59 @@ The following sections describe attributes, child elements, and parent elements.
+| \*\*Sequence\*\* | Optional \*\*Integer\*\*. Specifies the ordering priority to use for the list template on the \*\*Create\*\* page. If \*\*Sequence\*\* is not set, the template appears last in arbitrary order with any other templates that also lack a \*\*Sequence\*\* value. Two templates that specify the same sequence are sorted together in an arbitrary order. |
+| \*\*SetupPath\*\* | Optional \*\*Text\*\*. Specifies the path to a folder in the SharePoint Foundation setup directory (%ProgramFiles%\\Common Files\\Microsoft Shared\\web server extensions\\15\\\TEMPLATE) that contains a file to include in the template. |
+| \*\*SyncType\*\* | Optional \*\*Boolean\*\*. \*\*TRUE\*\* to override the default client list type associated with a SharePoint Foundation base template. |
+| \*\*Type\*\* | Optional \*\*Integer\*\*. Provides a unique identifier for the template. This identifier must be unique within the feature, but need not be unique across all feature definitions or site definitions. For a list of the default list template types, see SPListTemplateType.<br><br>This attribute corresponds to the \*\*Type\*\* attribute of the [List](list-element-list.md) element. |
+| \*\*Unique\*\* | Optional \*\*Boolean\*\*. \*\*TRUE\*\* to specify that the list definition or list template can be used to create a list only during site creation and cannot be used to create a list through the object model or user interface after the site template or definition is applied. Setting this attribute to \*\*TRUE\*\* has the effect of making the list hidden so that it does not appear as an option on the \*\*Create\*\* page or on the \*\*Documents and Lists\*\* page. |
+| \*\*UseRootFolderForNavigation\*\* | Optional \*\*Boolean\*\*. \*\*TRUE\*\* to specify that a link to the list that is displayed in \*\*Quick Launch\*\* points to the root folder so that users go to a custom welcome page, instead of to the default list view page.<br><br>If a custom welcome URL is not set for the list, rendering a link to the root folder is functionally equivalent to linking to the default list view. |
+| \*\*VersioningEnabled\*\* | Optional \*\*Boolean\*\*. \*\*TRUE\*\* to specify that versioning is enabled by default in lists created through the template; otherwise, \*\*FALSE\*\*. |
 
 ### Child elements
 
 None
 
 ### Parent elements
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p><a href="elements-element-list-template.md">Elements</a></p></td>
-</tr>
-</tbody>
-</table>
-
+[Elements](elements-element-list-template.md)
```

Comment:
We've been directed to keep Markdown tables as tables. You can use "Element" or "Elements" as the header, or if you find a more specific header that fits, use that.